File: finntech/src/main/java/com/example/finntech/entity/Cliente.java
package com.example.finntech.entity;

import jakarta.persistence.*;
import lombok.AccessLevel;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.ArrayList;
import java.util.List;
import java.util.UUID;

@Entity
@Table(name = "cliente")
@Getter @Setter @NoArgsConstructor
public class Cliente {
    @Id
    @GeneratedValue(strategy = GenerationType.UUID)
    @Setter(AccessLevel.NONE)
    private UUID id;
    private String nome;
    private String email;
    private String senha;
    private String cpf;

    @ManyToMany
    private List<Conta> contas = new ArrayList<>();
}

File: FinnTech-Front/src/app/model/classes/Cliente.ts
import { ICliente } from "../interfaces/ICliente";
import { IConta } from "../interfaces/IConta";

export class Cliente implements ICliente {
    id: String = "";
    nome: String = "";
    email: String = "";
    senha: String = "";
    cpf: String = "";
    contas: IConta[] = new Array<IConta>();
}